.semaphore/push_parametrised_versions.yml

```diff
@@ -0,0 +1,19 @@
+version: v1.0
+name: Push Parametrised versions to Dockerhub
```

Comment:
Please change the name here to something similar to this:
```
Push Image to DockerHub - elixir ${{parameters.ELIXIR_VERSION}}, protoc ${{parameters.PROTOC_VERSION}}, protobuf  ${{parameters.PROTOBUF_VERSION}}
```
so we can know which versions were pushed without checking the logs.

It will generate the name in the following format:
```
Push Image to DockerHub - elixir 1.11.4, protoc 3.17.3, protobuf  0.5.4
```

Everything else is :ok_hand:

Reply:
wow it works like a magic! tnx 🍨